src/common.ts

```diff
@@ -1,84 +1,74 @@
-import type { Client, OperationResult } from '@urql/core'
-import { atom } from 'jotai/vanilla'
+import type { Client, OperationContext, OperationResult } from '@urql/core'
+import { AnyVariables, Operation } from '@urql/core'
+import { atomWithObservable } from 'jotai/utils'
```

Comment:
```suggestion
import { atomWithObservable } from 'jotai/vanilla/utils'
```
It allows to use `jotai-urql` without React.